draft-packages/split-button/KaizenDraft/SplitButton/SplitButton.tsx

```diff
@@ -32,7 +32,7 @@ export interface SplitButtonProps
 /**
  * @deprecated Draft SplitButton is deprecated. Use "@kaizen/split-button" instead.
  */
-export const SplitButton: React.FunctionComponent<SplitButtonProps> = ({
+export const SplitButton: React.VFC<SplitButtonProps> = ({
```

Comment:
This is another one that's technically a breaking change, but I had a look through usages and no usages of children with SplitButton so should be good here as well 👍